<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1165</width>
    <height>781</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowOpacity">
   <double>30.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">

background-color: rgb(90, 90, 90);</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1161</width>
     <height>781</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>1165</width>
        <height>781</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(95, 95, 95);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>461</width>
         <height>681</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(95, 95, 95);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="processedImgLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>431</width>
          <height>381</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>BusyCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">
background-color: rgb(83, 82, 79);

border: 5px solid #ff2773</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="startButton">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>410</y>
          <width>241</width>
          <height>61</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>10</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	font: 75 16pt &quot;Rockwell&quot;;
color:rgb(95, 94, 97);
background-color: rgb(0, 0, 0);
border-radius : 30px;
}

QPushButton:hover{
color:rgb(147, 147, 147);
background-color: rgb(255, 244, 164);
}
QPushButton{
color:rgb(255, 39, 115);
}
</string>
        </property>
        <property name="text">
         <string>START</string>
        </property>
       </widget>
       <widget class="QPushButton" name="stopButton">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>490</y>
          <width>241</width>
          <height>61</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	font: 75 16pt &quot;Rockwell&quot;;
color:rgb(95, 94, 97);
background-color: rgb(0, 0, 0);
border-radius : 30px;
}

QPushButton:hover{
color:rgb(147, 147, 147);
background-color: rgb(255, 244, 164);
}
QPushButton{
color:rgb(255, 39, 115);
}
</string>
        </property>
        <property name="text">
         <string>STOP</string>
        </property>
       </widget>
       <widget class="QPushButton" name="closeButton">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>580</y>
          <width>241</width>
          <height>61</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	font: 75 16pt &quot;Rockwell&quot;;
color:rgb(95, 94, 97);
background-color: rgb(0, 0, 0);
border-radius : 30px;
}

QPushButton:hover{
color:rgb(147, 147, 147);
background-color: rgb(255, 244, 164);
}
QPushButton{
color:rgb(255, 39, 115);
}
</string>
        </property>
        <property name="text">
         <string>CLOSE</string>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>80</y>
         <width>641</width>
         <height>641</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(83, 82, 79);
border-radius : 30px;
border: 5px solid #ff2773</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="usrLabel">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>420</y>
          <width>531</width>
          <height>161</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>16</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 16pt &quot;Rockwell&quot;;
color:rgb(95, 94, 97);
background-color: rgb(0, 0, 0);
border: 1px solid;
 color:rgb(95, 94, 97);
border-radius : 30px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="text">
         <string>INITIALIZATING.......</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="margin">
         <number>20</number>
        </property>
        <property name="indent">
         <number>-1</number>
        </property>
       </widget>
       <widget class="QLabel" name="inLabel">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>120</y>
          <width>531</width>
          <height>171</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>16</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 16pt &quot;Rockwell&quot;;
color:rgb(95, 94, 97);
background-color: rgb(0, 0, 0);
border: 1px solid;
 color:rgb(95, 94, 97);
border-radius : 30px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="text">
         <string>PRESS START....</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
        </property>
        <property name="margin">
         <number>20</number>
        </property>
        <property name="indent">
         <number>-1</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>20</y>
          <width>90</width>
          <height>90</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>320</y>
          <width>90</width>
          <height>90</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
 background-immagesize: 90px 90px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <zorder>usrLabel</zorder>
       <zorder>inLabel</zorder>
       <zorder>label_5</zorder>
       <zorder>label_3</zorder>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>1151</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius : 30px;
background-color: rgb(95, 95, 95);

color: rgb(237, 240, 65);
font: 16pt &quot;Algerian&quot;;
border: 5px solid #edf041</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="text">
        <string>TOM CHAT BOT</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
